src/components/study-pane.js

```diff
@@ -391,6 +401,15 @@ const StudyPane = (props) => {
         }
     }
 
+    const checkStudyExists = useCallback(() => {
+        fetchStudy(studyUuid).then((study) => {
```

Comment:
fetchStudy Get the study if exists, it doesn't only check the study existance.
Maybie we should have a `fetchStudyExist` rest api function, which define `{method: HEAD}` when calling backendFetch ? (see recent conversations about checking element existancy with HEAD REST API method)